misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingOptions.java

```diff
@@ -24,16 +24,19 @@
     private int tabSize;
 
     // Prefer spaces over tabs.
-    private boolean insertSpaces;
+    private String wsCharacter;
+
+    private int columnLimit;
 
     FormattingOptions() {
         this.tabSize = 4;
-        this.insertSpaces = true;
+        this.wsCharacter = " ";
+        this.columnLimit = 80;
```

Comment:
Shall we change the default line length to 120? Since it is the limit specified in the current guide [1].

[1] https://ballerina.io/swan-lake/learn/coding-conventions/#indentation-and-line-length